versions/ufo3/glyphs/glif.md

```diff
@@ -167,6 +167,18 @@ This element may occur any number of times.
 
 #### This element has no child elements.
 
+#### Anchor naming conventions
+
+The following are agreed upon standard naming conventions for the `name` of anchors for special uses.
+
+##### Ligature Carets
+
+Anchors with a `name` starting with `caret_` or `vcaret_` indicate horizontal or vertical position for ligature glyphs, for example to compile the [OpenType GDEF Ligature Caret List table]. Anchors with the name `caret_` provide the `x` value for a ligature caret, and the `y` value is ignored. Likewise, anchors with the name `vcaret_` provide the `y` values for a ligature caret, and the `x` value is ignored.
+
+Both names may be followed arbitrary text (`caret_1`, `caret_blah`, etc) for designers or applications to have unique caret position names.
+
+There must be no more than N-1 caret postions for N ligature components.
```

Comment:
I would still say "There must be exactly N-1 caret positions for N ligature components." (also note the typo in positions)